lib/streamlit/DeltaGenerator.py

```diff
@@ -22,21 +22,32 @@
 setup_2_3_shims(globals())
 
 import functools
+import contextlib
+import traceback
+import threading
 import json
 import random
 import textwrap
+import re
 import pandas as pd
+import sys as _sys
```

Comment:
You can just import this as `sys` here. We imported things with an underscore before to hide it from users when autocompleting `st.<tab>`

Same below.